### PR TITLE
MATE 1.20 update December 2018

### PR DESCRIPTION
**Pull request by Mno-hime**
~~Upstream keeps adding new releases of the 1.20 stream, let's wait till they are done with it and merge afterwards.~~

I think they are done with dot-updates for now.

### CODE REVIEW

File: components/desktop/mate/mate-applets/Makefile [hunk 1]
```diff
@@ -76,8 +76,6 @@ REQUIRED_PACKAGES += library/glib2
 REQUIRED_PACKAGES += library/libgtop
 REQUIRED_PACKAGES += library/libnotify
 REQUIRED_PACKAGES += library/libxml2
-REQUIRED_PACKAGES += library/python/pygobject-3-27
```
Comment (pyhalov) [inline components/desktop/mate/mate-applets/Makefile:79]: Are you sure that this is not build deps?

Comment (Mno-hime) [reply components/desktop/mate/mate-applets/Makefile:79]: It actually is required, I brought it back. Thanks.

File: components/desktop/mate/mate-applets/Makefile [hunk 2]
```diff
@@ -62,7 +62,8 @@ install:	$(INSTALL_64)
 
 test:		$(NO_TESTS)
 
-REQUIRED_PACKAGES += desktop/character-map/gucharmap
```
Comment (pyhalov) [inline components/desktop/mate/mate-applets/Makefile]: gucharmap is still a build dependency.

Comment (Mno-hime) [reply components/desktop/mate/mate-applets/Makefile]: Now it's back.

**Pull request merged**